content/docs/guides/proxy-exporter.md

```diff
@@ -0,0 +1,375 @@
+---
+title: Understanding and using the proxy exporter pattern
+---
+
+# Understanding and using the proxy exporter pattern
```

Comment:
Proxy exporter is still not a term that we use blackbox-like is the closest.

Reply:
As per https://github.com/prometheus/docs/issues/1193#issuecomment-435324102 I would suggest `1:n exporter` - that way you can also directly explain why they need target info from their caller in the text below.

Reply:
I think we should have that discussion where everyone else is having it. In the issue.
Also please be careful with the "we".  I am sure you don’t want to imply that all the people who expressed different proposals are not part of the project.